src/CellData/CellFields.jl

```diff
@@ -242,16 +241,7 @@ end
 
 function return_cache(f::CellField,x::Point)
   trian = get_triangulation(f)
-  topo = GridTopology(trian)    # Note: this is expensive
-  vertex_coordinates = Geometry.get_vertex_coordinates(topo)
-  kdtree = KDTree(map(nc -> SVector(Tuple(nc)), vertex_coordinates))
-  D = num_cell_dims(trian)
-  vertex_to_cells = get_faces(topo,0,D)
-  cell_to_ctype = get_cell_type(trian)
-  ctype_to_reffe = get_reffes(trian)
-  ctype_to_polytope = map(get_polytope, ctype_to_reffe)
-  cell_map = get_cell_map(trian)
-  cache1 = kdtree, vertex_to_cells, cell_to_ctype, ctype_to_polytope, cell_map
+  cache1 = my_trian_data(trian)
```

Comment:
I would rather give a more meaningful name to `my_trian_data`, such as `point_to_cell_cache`... what do you think?